pytest_rts/utils/test_selection.py

```diff
@@ -1,4 +1,6 @@
-"""This module contains code for the main functionality of the RTS tool"""
+"""This module contains code for the test selection functionality of the RTS tool"""
+# pylint: disable=inherit-non-class
+# pylint: disable=too-few-public-methods
```

Comment:
Please don't disable those globally, add them to the lines where they are needed.